### PR TITLE
increase vcpu for s390x also; reduce schedule for image ecosystem, jenkins job

### PR DESCRIPTION
**Pull request by LakshmiRavichandran1**

### CODE REVIEW

File: ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.9.yaml [hunk 1]
```diff
@@ -241,7 +241,7 @@ tests:
     workflow: openshift-e2e-libvirt
 - as: ocp-image-ecosystem-remote-libvirt-ppc64le
   cluster: build01
-  cron: 0 1,13 * * *
+  cron: 0 1 * * *
```
Comment (deepsm007) [inline ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.9.yaml:244]: We do not want to change anything for ppc64le

Comment (LakshmiRavichandran1) [reply ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.9.yaml]: addressed in the recent commit

**Comment by deepsm007** [comment]
Can we drop down jenkins-e2e and image-ecosystem for 4.6, 4.7 & 4.8 to once a day?

**Comment by LakshmiRavichandran1** [comment]
for 4.6, 4.7, 4.8 it was already done by https://github.com/openshift/release/pull/23010/files

File: ci-operator/step-registry/ipi/install/libvirt/install/ipi-install-libvirt-install-commands.sh [hunk 2]
```diff
@@ -123,15 +123,26 @@ for ((i=0; i<${MASTER_REPLICAS}; i++))
...
 
+#if [ "${BRANCH}" == "4.9" ] || [ "${BRANCH}" == "4.10" ] && [ "${ARCH}" == "s390x" ]; then
+ #yq write --inplace ${dir}/openshift/99_openshift-cluster-api_worker-machineset-0.yaml spec.providerSpec.value[domainVcpu] 8
+#fi
+echo "BRANCH = ${BRANCH}"
+echo "ARCH = ${ARCH}"
+if [ "${BRANCH}" == "4.9" ] || [ "${BRANCH}" == "4.10" ] && [ "${ARCH}" == "s390x" ]; then
+  echo "condition met"
+fi
+yq write --inplace ${dir}/openshift/99_openshift-cluster-api_worker-machineset-0.yaml spec.providerSpec.value[domainVcpu] 8
```
Comment (deepsm007) [inline ci-operator/step-registry/ipi/install/libvirt/install/ipi-install-libvirt-install-commands.sh:143]: `spec.template.spec.providerSpec.value.`  for worker node

Comment (LakshmiRavichandran1) [reply ci-operator/step-registry/ipi/install/libvirt/install/ipi-install-libvirt-install-commands.sh]: resolved in the recent commit.

**Review by deepsm007 — APPROVED** [review]
Temporary bumping the grace_period and worker_node domainVcpu looks good to confirm the performance issues.

**Pull request merged**